openapi.yaml

```
@@ -13781,6 +13907,421 @@ components:
                 - data
                 - has_more
 
+        AuditLogActorServiceAccount:
+            type: object
+            description: The service account that performed the audit logged action.
+            properties:
+                id:
+                    type: string
+                    description: The service account id.
+
+        AuditLogActorUser:
+            type: object
+            description: The user who performed the audit logged action.
+            properties:
+                id:
+                    type: string
+                    description: The user id.
+                email:
+                    type: string
+                    description: The user email.
+
+        AuditLogActorApiKey:
+            type: object
+            description: The API Key used to perform the audit logged action.
+            properties:
+                id:
+                    type: string
+                    description: The tracking id of the API key.
+                type:
+                    type: string
+                    description: The type of API key. Can be either `user` or `service_account`.
+                    enum: ["user", "service_account"]
+                user:
+                    $ref: "#/components/schemas/AuditLogActorUser"
+                service_account:
+                    $ref: "#/components/schemas/AuditLogActorServiceAccount"
+
+        AuditLogActorSession:
+            type: object
+            description: The session in which the audit logged action was performed.
+            properties:
+                user:
+                    $ref: "#/components/schemas/AuditLogActorUser"
+                ip_address:
+                    type: string
+                    description: The IP address from which the action was performed.
+
+        AuditLogActor:
+            type: object
+            description: The actor who performed the audit logged action.
+            properties:
+                type:
+                    type: string
+                    description: The type of actor. Is either `session` or `api_key`.
+                    enum: ["session", "api_key"]
+                session:
+                    type: object
+                    $ref: "#/components/schemas/AuditLogActorSession"
+                api_key:
+                    type: object
+                    $ref: "#/components/schemas/AuditLogActorApiKey"
+                    
+                    
+        AuditLogEventType:
+            type: string
+            description: The event type.
+            x-oaiExpandable: true
+            oneOf:
+                - type: string
+                  title: api_key.created
```

note: this should be an enum
```suggestion
        AuditLogEventType:
            type: string
            description: The event type.
            x-oaiExpandable: true
            enum:
              - `api_key.created`
              - ...
```
otherwise we generate code like this:
```py
Union[str, str, str, ...]
```

---

will make that change! I had been adjusting the schema because the UI wasn't showing the enum values, but that made it less accurate.